service/application/src/main/java/com/solace/maas/ep/event/management/agent/command/CommandManager.java

```diff
@@ -45,14 +48,14 @@ public class CommandManager {
     private final MessagingServiceDelegateService messagingServiceDelegateService;
     private final EventPortalProperties eventPortalProperties;
     private final ThreadPoolTaskExecutor configPushPool;
-    private final CommandLogStreamingProcessor commandLogStreamingProcessor;
+    private final Optional<CommandLogStreamingProcessor> commandLogStreamingProcessorOpt;
```

Comment:
commandLogStreamingProcessor is absent for cloud managed mode. So making it optional